src/components/ContentUploader/ContentUploader.js

```diff
@@ -419,6 +430,7 @@ class ContentUploader extends Component<Props, State> {
         });
 
         this.updateViewAndCollection(items);
+        this.upload();
```

Comment:
Same question - did you mean to add this?

Reply:
yes, failed upload makes room for another item. Calling upload() to scan the items and find the next pending items to upload.

Reply:
Got it